### PR TITLE
F4 stop & Standby are working

### PR DESCRIPTION
**Pull request by NMSTEC**

### CODE REVIEW

**Comment by NMSTEC** [comment]
#3753 #3660

File: firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp [hunk 1]
```diff
@@ -186,8 +186,8 @@ void stm32_stop() {
 	enginePins.warningLedPin.setValue(0);
 
 	// Do anything the board wants to prepare for stop mode - enabling wakeup sources!
-	boardPrepareForStop();
 
+	boardPrepareForStop();
```
Comment (mck1117) [inline firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp:190]: huh?

Comment (NMSTEC) [reply firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp:190]: Commented in Slack. Accidently just moved that line down while comparing files. Does not need changing, and in reality nothing changed.

Comment (mck1117) [reply firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp:190]: can you undo the change to reflect that nothing changed?

File: firmware/hw_layer/ports/stm32/stm32f4/mpu_util.cpp [hunk 2]
```diff
@@ -4,9 +4,10 @@
  * @date Jul 27, 2014
  * @author Andrey Belomutskiy, (c) 2012-2020
  */
-
+#include "pch.h"
 #include "flash_int.h"
 
+
```
Comment (mck1117) [inline firmware/hw_layer/ports/stm32/stm32f4/mpu_util.cpp:10]: same here with an extra line

File: firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp (hunk 1 above)
Comment (mck1117) [reply firmware/hw_layer/ports/stm32/stm32f7/mpu_util.cpp:189]: line got deleted still

**Comment by NMSTEC** [comment]
F407 Does not have LPUDS or UDEN registers.

**Review by rusefillc — CHANGES_REQUESTED** [review]
this PR updates wideband submodule which it should not be doing

this PR would need to be re-done or i can pick pieces manually once it's green

**Comment by rusefillc** [comment]
@NMSTEC let's make it F42x conditional and green

**Comment by NMSTEC** [comment]
> @NMSTEC let's make it F42x conditional and green

Thats where the problem lay. It's failing only with F40X MCUs. You guys only defined F4XX & F7XX. Would require redefining that.

**Comment by mck1117** [comment]
you can check `#ifdef STM32F429xx`

**Comment by rusefillc** [comment]
@NMSTEC there is no "you guys" - you are the boss of this PR

**Pull request closed without merging**